.rubocop.yml

```diff
@@ -46,12 +46,15 @@ Style/FormatString:
 Style/RaiseArgs:
   Enabled: false
 
-Naming/UncommunicativeMethodParamName:
+Naming/MethodParameterName:
   Enabled: false
 
 Metrics:
   Enabled: false
 
+Layout/LineLength:
```

Comment:
I'm not sure we should disable the line length, maybe we limit it instead.

Reply:
Let me put it this way: Before, this was `Metrics/LineLength` and thus disabled with `Metrics:` alltogether. :)